src/Fields/field_tuples.jl

```diff
@@ -29,24 +35,31 @@ function VelocityFields(arch, grid, bcs::NamedTuple)
     return (u=u, v=v, w=w)
 end
 
+
+#####
+##### Tracer fields tuples
+#####
+
 """
-    TracerFields(arch, grid, tracer_names; kwargs...)
+    TracerFields(tracer_names, arch, grid; kwargs...)
 
 Return a NamedTuple with tracer fields specified by `tracer_names` initialized as
 `CellField`s on the architecture `arch` and `grid`. Fields may be passed via optional
 keyword arguments `kwargs` for each field.
 
+This function is used by OutputWriters.Checkpointer.
+
 # Examples
 ```julia
 arch = CPU()
 topology = (Periodic, Periodic, Bounded)
 grid = RegularCartesianGrid(topology=topology, size=(16, 16, 16), size=(1, 2, 3))
 tracers = (:T, :S, :random)
 noisy_field = CellField(arch, grid, TracerBoundaryConditions(grid), randn(16, 16))
-tracer_fields = TracerFields(arch, grid, tracers, random=noisy_field)
+tracer_fields = TracerFields(tracers, arch, grid, random=noisy_field)
 ```
```

Comment:
Should we convert to a `jldoctest`?

Not high priority though, I know it's kinda annoying sometimes. We could leave it for a big future PR that updates docs.

Reply:
Should probably remove the example if this is not an exported function.

Reply:
I removed it since doctests do take time so we should allocate resources where it matters. For devs the source is good!